eth/src/main/java/contract/BasicCoin.java

```diff
@@ -91,16 +91,16 @@ protected BasicCoin(String contractAddress, Web3j web3j, TransactionManager tran
     public RemoteCall<TransactionReceipt> approve(String _spender, BigInteger _value) {
         final Function function = new Function(
                 FUNC_APPROVE, 
-                Arrays.<Type>asList(new org.web3j.abi.datatypes.Address(_spender), 
-                new org.web3j.abi.datatypes.generated.Uint256(_value)), 
+                Arrays.<Type>asList(new Address(_spender),
```

Comment:
This file is autogenerated code, do not modify this, please.

Reply:
Sure I will rollback the changes. Probably it was done by IDE automatically.